### PR TITLE
test: ✅ passed the first unit test for slog logging info

### PR DESCRIPTION
**Pull request by Sajal008**
This is a unit test for slog logging info that need review.

### CODE REVIEW

File: tests/test_slogpy_console.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""Import slogpy from slog"""
+from slogpy import slog
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: This will not work as you think it does.  Please make sure to run the code before submitting a PR.

Comment (neel-m) [reply tests/test_slogpy_console.py]: @Sajal008 It looks like you addressed the issue.  In the future when you address an issue, please resolve the conversation.  It makes it easier to keep track of what is left to do.

File: tests/test_slogpy_console.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"""Import slogpy from slog"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: suggestion: this doesn't seem like a docstring for the module (which is what its location implies). 
PEP 257 describes docstrings in detail: https://peps.python.org/pep-0257/

File: tests/test_slogpy_console.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: suggestion:  If you feel each function needs more description, please use the multiline format for the docstring.

File: tests/test_slogpy_console.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+def setupfunction():
+    """Not to create a file for output"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: nitpick: this is a very awkward syntax, also this is the place to say why you made this choice (the choice you made is good)

File: tests/test_slogpy_console.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    slog.initialize(file_logging=False)
+
+"""First unit test"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: suggestion: this seems unnecessary
nitpick: this seems like an unusual use of the triple double quote
pep 257 talks about docstrings (it is worth reading) https://peps.python.org/pep-0257/
pep 8 is the basic style guide (also worth reading): https://peps.python.org/pep-0008/
single line comments normally use the '#' character

File: tests/test_slogpy_console.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+def test_info_console_output(capsys, msg="This is a console test"):
+    """Passing a message to the info method and then capture the message from the console"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py]: suggestion: using the Given-When-Then syntax will make the purpose of this test clearer ex:
Given: using slogpy
When: use info() with a string parameter
Then: the string is output to the console AND nothing is output to stderr

**Review by neel-m — CHANGES_REQUESTED** [review]
Looking good

File: tests/test_slogpy_console.py (hunk 2 above)
Comment (Sajal008) [reply tests/test_slogpy_console.py]: Understood, I will try my best to follow the protocols of docstring and describe the module's purpose and functionality.

File: tests/test_slogpy_console.py (hunk 3 above)
Comment (Sajal008) [reply tests/test_slogpy_console.py]: I will review the docstring one more time and will also make more use of multiline format. Thanks.

File: tests/test_slogpy_console.py (hunk 6 above)
Comment (Sajal008) [reply tests/test_slogpy_console.py]: I have started using this syntax. It makes the code more informative. This is a game changer. Thank you.

File: tests/test_slogpy_console.py (hunk 2 above)
Comment (neel-m) [reply tests/test_slogpy_console.py]: do not resolve a conversation until the code is changed (in the PR) or not changing it has been discussed.

File: tests/test_slogpy_console.py (hunk 3 above)
Comment (neel-m) [reply tests/test_slogpy_console.py]: do not resolve a conversation until the code is changed (in the PR) or not changing it has been discussed.

File: tests/test_slogpy_console.py (hunk 5 above)
Comment (neel-m) [reply tests/test_slogpy_console.py]: do not resolve a conversation until the code is changed (in the PR) or not changing it has been discussed.

File: tests/test_slogpy_console.py (hunk 6 above)
Comment (neel-m) [reply tests/test_slogpy_console.py]: do not resolve a conversation until the code is changed (in the PR) or not changing it has been discussed.

**Pull request closed without merging**